src/components/Tips/Tip.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC, PropsWithChildren } from 'react'
+
+interface TipProps {
+  display: boolean
+}
+
+/** A tip that gets displayed at the bottom of the window. */
+const Tip: FC<PropsWithChildren<TipProps>> = ({ display, children }, ref) => {
+  return (
+    <div
+      style={{
+        position: 'fixed',
+        bottom: '1em',
+        left: 0,
+        width: '100%',
+        zIndex: 20,
```

Comment:
You can give it the `.z-index-popup` class:

https://github.com/cybersemics/em/blob/aab54a042c2fb83c2dd0b8141fe5fe46c8fc7ecb/src/App.css#L1-L5

https://github.com/cybersemics/em/blob/aab54a042c2fb83c2dd0b8141fe5fe46c8fc7ecb/src/App.css#L43-L45

I like to keep all my zIndexes in one place to avoid conflicts and make it easier to introduce new layers.

Reply:
Implemented here: 61f6b011025db55c048333d1f8224d4dca8085ee